maestro-client/src/main/java/maestro/drivers/IOSDriver.kt

```diff
@@ -195,6 +195,8 @@ class IOSDriver(
                     "architecture where view flattening is available. For more information on the " +
                     "migration process, please visit: https://reactnative.dev/docs/new-architecture-intro"
             Insights.report(Insight(message, Insight.Level.INFO))
+        } else {
+            Insights.report(Insight("", Insight.Level.NONE))
```

Comment:
This API still feels weird to me since we're emitting an empty insight on every hierarchy call. I understand that this allows us to clear the banner message, but it doesn't make sense in the context of the terminal, and also doesn't makes sense if we ever introduce any other insights. I'm fine shipping with this as a initial implementation but I think we're going to have to rethink how this is architected as soon as we want to do anything else with the Insights class.

Reply:
Why does this doesn't fit on terminal. We are filtering on terminal side as well on the basis of level of insight in the same way how we do it for studio 🤔 

Do you mean we need more intuitive API to clear the current shown insight irrespective its studio or CLI?

Reply:
I mean in the terminal does `Insights.report(Insight("", Insight.Level.NONE))` do anything or is it a no-op?